packages/ffe-context-message-react/src/ContextErrorMessage.js

```diff
@@ -16,14 +17,14 @@ const ContextErrorMessage = props => {
     return (
         <ContextMessage
             {...rest}
+            locale={locale}
             messageType="error"
-            aria-label="Feilmelding"
```

Comment:
Disse har jag fjernet konsekvent i komponenterna for dom er ikke i bruk. En alert skall vel heller kanske ikke ha en aria label?

Reply:
Det er jeg ikke sikker på stemmer?  Vi har UU testet disse komponentene og da funket det bra å ha label på gruppe/alert nivået.  Så per nå er jeg litt uenig i at aria-labelene settes på ikonene istedenfor.  Her kan man kanskje høre med Sindre om best practice

Reply:
Kort svar: 
Det kommer an på konteksten man implementerer elementet i. Om ikonet står for seg selv vil man måtte legge til informasjon i koden, men om det er supplerende skriftlig informasjon sammen med ikonet kan man like gjerne klassifisere ikonet som dekorativt. 
Å gi alle brukere jevnverdig god informasjon på en brukervennlig måte er det viktigste.

Langt svar:
Aria-label skal som første regel kun benyttes når det ikke er mulig å bruke HTML-attributter for å gi ekstra kontekstuell informasjon. Så å bruke aria-label i stedet for HTML-attributter for disse elementene vil da være riktig.

Når det kommer til uu-hensyn kommer det i dette tilfellet an på kontekst for elementet. 

Om ikonet presenteres med skriftlig beskjed om hva som har skjedd (som for eksempel "Betalingen ble registrert!") kan man argumentere for at ikonet er supplerende informasjon og derfor ikke trenger å bli plukket opp av skjermlesere i det hele tatt.

Om den tiltenkte konteksten er å benytte elementet for en alert er det viktig at dette gjøres korrekt. Om alerten krever å raskt kunne informere brukeren vil det være viktig å sørge for at dette ivaretas for alle brukere, om nødvendig med ARIA: alert.

Reply:
Når jag googlet så sa jag flere exempler med aria label på group men ikke direkt på alert. Jag tolkat koden slik att aria lablen aldrig ble brukt nedøver men ser nå den ligger på {...rest}. Har tagit tilbake den nå iaf. Hensikten var ikke or ersette med ikonet. Ikonet synes jag dermot er øverflødig ;)

Reply:
Du har helt rett i at label kan brukes på gruppert informasjon :)

Som nevnt er det 100% avhengig av konteksten innholdet settes i, også for om ikonet skal ha label eller ikke